rollup/sync_service/sync_service.go

```diff
@@ -237,6 +259,16 @@ func (s *SyncService) fetchMessages() {
 			return
 		}
 
+		// write start index of very first L1MessageV2 to database. This is true only once.
+		if len(msgsV2) > 0 && l1MessageV2StartIndex == nil {
+			firstL1MessageV2 := msgsV2[0]
```

Comment:
Worth adding an info log here

Reply:
done in c72cbfcf4c43502c393ac3f45e9a6d8a5578b261